code/modules/chemistry/tools/bottles.dm

```diff
@@ -198,77 +199,77 @@
 
 /obj/item/reagent_containers/glass/bottle/sulfonal
 	name = "sulfonal bottle"
-	desc = "A small bottle."
+	desc = "A small bottle of sulfonal, an extreme muscle relaxant. It has a warning label on it about impaired breathing and drowsiness."
 	bottle_style = "4"
 	amount_per_transfer_from_this = 5
 	initial_reagents = "sulfonal"
 
 /obj/item/reagent_containers/glass/bottle/synaptizine
 	name = "bottle (synaptizine)"
-	desc = "A small bottle."
+	desc = "A small bottle of synaptizine, a non-addictive stimulant whose side effects can cause regeneration of nerve cells."// pulled this one outta my ass but it's important to convey it's brain medicine
```

Comment:
```suggestion
	desc = "A small bottle of synaptizine, a non-addictive stimulant whose side effects can cause regeneration of brain tissue."
```
removed comment as its more directed at the PR, modified description to be more brain-centric as opposed to more general (nervous system)